src/Passes/QirExamples/Bernstein–Vazirani/QSharpVersion/Example.qs

```diff
@@ -0,0 +1,108 @@
+namespace Tets.Samples.BernVaz {
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Measurement;
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Convert;
+
+    @EntryPoint()
+    operation BernsteinVaziraniTestCase (nQubits : Int, patternInt : Int) : Result[] {
```

Comment:
This operation is named as a function (that is, as a noun rather than as a verb).